main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -3325,6 +3325,7 @@ object Resolver {
         case TypeConstructor.CaseUnion(_) => Result.Err(ResolutionError.IllegalType(tpe, loc))
         case TypeConstructor.Error(_) => Result.Err(ResolutionError.IllegalType(tpe, loc))
 
+        case TypeConstructor.AnyType => throw InternalCompilerException(s"unexpected type: $tc", tpe.loc)
```

Comment:
Move upwards

Reply:
You mean remove the empty line above or move anytype up to unit? Both of them is inconsistent with the rest of the function.